src/test/java/hudson/plugins/clover/RatioTest.java

```diff
@@ -76,4 +76,48 @@ public void testGetPercentage() {
         assertEquals("1000/1000 => 100", 100, Ratio.create(1000,1000).getPercentage());
     }
 
+    /**
+     * Tests that {@link Ratio#getPcCovered()} correctly handles values near
+     * whole integers.  This is to make sure that it will match that of
+     * getPercentage()
+     */
+    public void testGetPcCovered() {
+        // Make sure zero is zero
+        assertEquals("0/10000     =>    0",    "0%", Ratio.create(    0, 10000).getPcCovered());
+        assertEquals("9/10000     =>    0",    "0%", Ratio.create(    9, 10000).getPcCovered());
+
+        assertEquals("99/10000    =>  0.9",  "0.9%", Ratio.create(   99, 10000).getPcCovered());
+
+        assertEquals("9009/10000  =>   90",   "90%", Ratio.create( 9009, 10000).getPcCovered());
+        assertEquals("9099/10000  => 90.9", "90.9%", Ratio.create( 9099, 10000).getPcCovered());
+
+        assertEquals("9909/10000  =>   99",   "99%", Ratio.create( 9909, 10000).getPcCovered());
```

Comment:
I think it would be better to always have one fraction digit, e.g. "99.0%" and "99.9%". It will look more consistent.